utils/obsproc/Rads2Ioda.h

```diff
@@ -57,10 +57,31 @@ namespace gdasapp {
       ncFile.getVar("time_mjd").getVar(datetime);
       iodaVars.referenceDate_ = "seconds since 1858-11-17T00:00:00Z";
 
+      std::map<std::string, int> altimeterMap;
```

Comment:
why is this not a function anymore

Reply:
So the map can be converted to a string that can be assigned to a IodaVar member

Reply:
It winds up taking less lines just to declare, assign, and use the map in-line